__tests__/app/models/session.jest.js

```diff
@@ -0,0 +1,30 @@
+const db = require('../../../app/models/index')
+const moment = require('moment')
+
+describe("Session model", () => {
+    let Session
+
+    beforeAll(async() => {
+        await db.sequelize.sync() // connect to the database
```

Comment:
This shouldn't be needed for the tests anymore - I made an initialization script that automatically connects to the database before any tests are run :)